app/models/case_log.rb

```diff
@@ -112,6 +112,10 @@ def in_progress?
     status == "in_progress"
   end
 
+  def supported_housing_schemes_enabled?
```

Comment:
I wonder if using `method_missing` and delegating might make this more generically useful... (or more complicated)

Reply:
I might need some more explanation around this as I'm not 100% sure what you mean